crates/cli/src/subcommands/call.rs

```diff
@@ -219,8 +219,7 @@ fn add_reducer_ctx_to_err(error: &mut String, schema_json: Value, reducer_name:
         .map(|kv| kv.0)
         .collect::<Vec<_>>();
 
-    // Hide pseudo-reducers (assume that any `__XXX__` are such); they shouldn't be callable.
-    reducers.retain(|&c| !(c.starts_with("__") && c.ends_with("__")));
+    // TODO(noa): exclude lifecycle reducers
```

Comment:
this is a (very minor) regression, so I just want to double-check that we're unblocking somebody/something here - is this PR part of the work for https://github.com/clockworklabs/SpacetimeDB/issues/1891 ?

Reply:
Correct. Ideally we would change the api format before 1.0 to fix this, but it'll still block a user from calling a lifecycle reducer even if it's not perfect dx